### PR TITLE
Anonymize simple

### PR DESCRIPTION
**Pull request by ML-engineer**

### CODE REVIEW

File: anonymize/src/main/java/org/datahem/processor/anonymize/AnonymizeStreamPipeline.java [hunk 1]
```diff
@@ -335,12 +313,11 @@ public static void main(String[] args) throws IOException {
 					new SerializableFunction<String, String>() {
 						@Override
 						public String apply(String tableSpec) {
-							return tableSpec.replaceAll("[^A-Za-z0-9.]", "");
+							return tableSpec.replaceAll("[^A-Za-z0-9._]", "");
 						}
```
Comment (draimosson-mathem) [inline anonymize/src/main/java/org/datahem/processor/anonymize/AnonymizeStreamPipeline.java:317]: Sorry for being slow, but what do these lines actually do?

Comment (draimosson-mathem) [reply anonymize/src/main/java/org/datahem/processor/anonymize/AnonymizeStreamPipeline.java:317]: It looks like it clears some fields in rows that are to be written to recovery table. Doesn't that defeat the purpose of the recovery table?

**Review by draimosson-mathem — COMMENTED** [review]
Great job! Just one question.

File: anonymize/src/main/java/org/datahem/processor/anonymize/AnonymizeStreamPipeline.java (hunk 1 above)
Comment (ML-engineer) [reply anonymize/src/main/java/org/datahem/processor/anonymize/AnonymizeStreamPipeline.java:317]: it just removes characters that are not allowed in a bigquery tablename, it has nothing to do with the actual data. It's just a safety measure if user provides a table name that isn't valid.

Comment (draimosson-mathem) [reply anonymize/src/main/java/org/datahem/processor/anonymize/AnonymizeStreamPipeline.java:317]: Ok, fine then!

**Pull request merged**